app/helpers/react_components/mentoring/discussion.rb

```diff
@@ -45,6 +45,7 @@ def iterations
             idx: iteration.idx,
             num_comments: ccs.sum(&:second),
             unread: ccs.reject { |(_, seen), _| seen }.present?,
+            created_at: iteration.created_at.iso8601,
```

Comment:
Where is this used in the UI? Just wondering if it should be created_at, or updated_at

Reply:
I've moved this comment to the other PR.